modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4333,6 +4333,7 @@ create-test=Create Test
 create-test-error=An unexpected error occurred while creating the A/B test.
 create-test-help-message=Create a new A/B test to optimize your page or experience.
 create-the-condition-of-the-action-using-the-expression-builder-type-x-to-use-the-autocomplete-feature=Create the condition of the action using the expression builder. Type {0} to use the autocomplete feature.
+create-the-trigger-of-the-action-using-expression-builder-type-x-to-use-the-autocomplete-feature=Create the trigger of the action using Expression Builder. Type {0} to use the autocomplete feature.
```

Comment:
@danvb112 As I pointed in other comment, we cannot use this sentence because we're not providing the autocomplete feature.